controller/src/beerocks/master/beerocks_master_main.cpp

```diff
@@ -218,6 +218,33 @@ static void fill_master_config(son::db::sDbMasterConfig &master_conf,
     } else {
         master_conf.load_steer_on_vaps = std::string(load_steer_on_vaps);
     }
+
+    if (!beerocks::bpl::cfg_get_persistent_db_enable(master_conf.persistent_db)) {
+        LOG(DEBUG) << "failed to read persistent db enable, setting to default value: "
+                   << bool(DEFAULT_PERSISTENT_DB);
+        master_conf.persistent_db = (DEFAULT_PERSISTENT_DB == 1);
```

Comment:
Why do you do that? DEFAULT_PERSISTENT_DB  defined as zero, it will always be false.

Reply:
you are right - I meant to write `bool(DEFAULT_PERSISTENT_DB)`

I will fix it